Tests/Validator/InlineValidatorTest.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\CoreBundle\Tests\Validator;
+
+use Sonata\CoreBundle\Tests\Fixtures\Bundle\Validator\FooValidatorService;
+use Sonata\CoreBundle\Validator\ErrorElement;
+use Sonata\CoreBundle\Validator\InlineValidator;
+use Symfony\Component\Validator\Exception\ValidatorException;
+
+/**
+ * Test for InlineValidator.
```

Comment:
For me this comment is useless.

By `class InlineValidatorTest`, we know this class is to test `InlineValidator`. :wink: